src/components/Header/MobileHeader/DrawerContent.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './DrawerContent.scss';
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemText
+} from "@material-ui/core";
+
+function ListItemLink(props) {
+  return <ListItem button component="a" {...props} />;
+}
+
+const MenuListItems = [
```

Comment:
Change the name to `menuListItems` because this one creates confusion. Component names should be start with upper case and variable names should start with lower case letters.

Reply:
Done